htdocs/install/mysql/tables/llx_asset.sql

```diff
@@ -15,20 +15,37 @@
 
 
 CREATE TABLE llx_asset(
-	rowid integer AUTO_INCREMENT PRIMARY KEY NOT NULL,
-	ref varchar(128) NOT NULL,
-	entity integer DEFAULT 1 NOT NULL,
-	label varchar(255),
-	amount_ht double(24,8) DEFAULT NULL,
-	amount_vat double(24,8) DEFAULT NULL,
-	fk_asset_type integer NOT NULL,
-	description text,
-	note_public text,
-	note_private text,
-	date_creation datetime NOT NULL,
-	tms timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
-	fk_user_creat integer NOT NULL,
-	fk_user_modif integer,
-	import_key varchar(14),
-	status integer NOT NULL
+	rowid						integer			AUTO_INCREMENT PRIMARY KEY NOT NULL,
+	entity						integer			DEFAULT 1 NOT NULL,  -- multi company id
+	ref							varchar(128)	DEFAULT '(PROV)' NOT NULL,
+	label						varchar(255)	NOT NULL,
+	fk_asset_model				integer,
+
+	reversal_amount_ht			double(24,8),
+	acquisition_value_ht		double(24,8)	NOT NULL,
+	recovered_vat				double(24,8),
+	reversal_date				date,
+	date_acquisition			date 			NOT NULL,
+	date_start					date 			NOT NULL,
+	qty							real			DEFAULT 1 NOT NULL,	-- qty asset
+	acquisition_type			smallint		DEFAULT 0 NOT NULL,	-- new, occasion
+	asset_type					smallint		DEFAULT 0 NOT NULL,	-- intangible, tangible, in progress, financial
+	not_depreciated				integer(1)		DEFAULT 0,
+
+	disposal_date				date,
+	disposal_amount_ht			double(24,8),
+	fk_disposal_type			integer,
+	disposal_depreciated		integer(1)		DEFAULT 0,
```

Comment:
integer(1) is not allowed into sql (pb of partability with mysql version)
use just
integer